homeassistant/components/forked_daapd/media_player.py

```diff
@@ -669,10 +669,45 @@ async def async_play_media(
 
         if media_type == MediaType.MUSIC:
             media_id = async_process_play_media_url(self.hass, media_id)
+        elif media_type not in CAN_PLAY_TYPE:
+            _LOGGER.warning("Media type '%s' not supported", media_type)
+            return
 
-            await self._async_announce(media_id)
-        else:
-            _LOGGER.debug("Media type '%s' not supported", media_type)
+        if kwargs.get(ATTR_MEDIA_ANNOUNCE):
+            return await self._async_announce(media_id)
+
+        if (
+            enqueue := kwargs.get(ATTR_MEDIA_ENQUEUE, MediaPlayerEnqueue.REPLACE)
+        ) is True:
+            enqueue = MediaPlayerEnqueue.ADD
+        if enqueue in {MediaPlayerEnqueue.ADD, MediaPlayerEnqueue.REPLACE}:
```

Comment:
Would this be clearer?
```suggestion
        enqueue: bool | MediaPlayerEnqueue | None = kwargs.get(ATTR_MEDIA_ENQUEUE)
        if enqueue in {None, True, MediaPlayerEnqueue.ADD, MediaPlayerEnqueue.REPLACE}:
```

Reply:
I actually think the original way is clearer because it is more clear about how `True` defaults to `MediaPlayerEnqueue.ADD`
However, I do think we can be clearer that what's stored in ATTR_MEDIA_ENQUEUE can be a bool or one of those enums.
Let me push a riff on your change with a comment to make the default behavior clearer.